irohad/main/flags.hpp

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_FLAGS_HPP_
+#define IROHA_FLAGS_HPP_
+
+#include <CLI/CLI.hpp>
+#include "main/common.hpp"
+#include "main/env-vars.hpp"
+#include "util/network.hpp"
+
+using namespace std::literals::string_literals;
+
+inline void addPeerFlags(CLI::App *p,
```

Comment:
I marked inline not because of code inlining, but because definition and implementation are in hpp and it is mandatory there.

Reply:
Yep, I know, that's why I suggest to move it to __.cpp__

Reply:
If I  move them to cpp, I should create new target, which I don't want to do. 
Current design for config is not scalable, it will be hard to manage configs. For now, it is okay, but in future should be replaced.